libfreerdp/core/test/TestFuzzCryptoCertificateReadServerCertificate.c

```diff
@@ -0,0 +1,9 @@
+#include <freerdp/crypto/certificate.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	rdpCertificate* certificate = NULL;
+	certificate_read_server_certificate(certificate, (BYTE*)Data, Size);
```

Comment:
invalid test case, `certificate` must not be `NULL`. (you might want to add some `WINPR_ASSERT` in `certificate_read_server_certificate` function though)

Reply:
never mind the `WINPR_ASSERT`, I´ll do a pr for that.